src/gui/folderman.cpp

```diff
@@ -825,30 +828,47 @@ void FolderMan::slotStartScheduledFolderSync()
 
 void FolderMan::slotEtagPollTimerTimeout()
 {
-    ConfigFile cfg;
-    auto polltime = cfg.remotePollInterval();
+    // Some folders need not to be checked because the use the new push notifications
+    QList<Folder *> foldersToRun;
+    for (auto folder : _folderMap) {
```

Comment:
This loop is std::copy_if really. Please favor STL algorithms to hand rolled loops.